igor-web/src/main/java/com/netflix/spinnaker/igor/scm/stash/client/model/DirectoryChild.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.igor.scm.stash.client.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.Data;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class DirectoryChild {
```

Comment:
Can you please add comments on what is DirectoryChild and DirectoryChildren?

Reply:
These are just representations of the elements in the response from the stash API (which are not documented BTW). Children is a collection of “childs” basically. I can add some comments.